public/app/features/explore/hooks/useStateSync/index.ts

```diff
@@ -343,6 +353,7 @@ const isFulfilled = <T>(promise: PromiseSettledResult<T>): promise is PromiseFul
 /**
  * Compare 2 explore urls and return a map of what changed. Used to update the local state with all the
  * side effects needed.
+ * TODO: this should also handle panelsState changes
```

Comment:
Can be removed now?

Reply:
lol you made the same comment in the previous pr and i must have resolved the conflict by putting it back

Reply:
ce71e5475e